pyvista/plotting/plotter.py

```diff
@@ -5631,10 +5631,12 @@ def save_graphic(self, filename, title='PyVista Export', raster=True, painter=Tr
         --------
         >>> import pyvista
         >>> from pyvista import examples
-        >>> pl = pyvista.Plotter()
+        >>> pl = pyvista.Plotter(off_screen=True)
```

Comment:
I'm hoping this `off_screen` argument isn't necessary and that we are setting `pyvista.OFF_SCREEN` when running the docstrings.

Reply:
Good point. thanks!